study.md

```diff
@@ -22,5 +22,11 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 In your own words, describe why someone would use Ember.
 
 ```md
-<!-- your answer here -->
+There are many reasons to use ember. They have built in best practices, such as issuing warning when practices has changed. Ember keeps your tech stack consistent and coherent, which improves the efficiency of moving developers in between project. In addition ember has amazing features, such as organizing your code with modules, in the ember-cli- the command line interface for ember. Ember CLI provides tools to enhance productivity, e.g. brocolli.js (which concatenate and minify your code. It even has testing built into it.
+
+There are so many amazing features in ember. It gives you URLs and a functioning back button by default with every route you make. The object system in Ember  takes the best part of ruby objects. Additionally they have computed properties.
```

Comment:
Yes, because Ember has URLs associated with each page that it shows, the browser can go 'back' to a previous page just as if you were working with static pages in a website.